sdk/advisor/arm-advisor/package.json

```diff
@@ -2,11 +2,11 @@
   "name": "@azure/arm-advisor",
   "author": "Microsoft Corporation",
   "description": "AdvisorManagementClient Library with typescript type definitions for node.js and browser.",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
Please bump a major version as the dependencies have updated which contains breaking changes.

Reply:
done